### DISCUSSION TITLE
I will get more donations and funding for the project if I can know a little more about the creator of the project

### DISCUSSION THREAD

**Discussion by Jakhotiya** [NoiseByNorthwest/php-spx#242] [General]
HI @NoiseByNorthwest , @AlexNDRmac 

My name is Abhishek. I am from india and have been using this tool for 1.5 years now. 

Recently I gave a talk on SPX and it's value as a profiler in meet magento india 2024. 800+ developers attended this.
I showed how this is an invaluable tool for the developer. My presentation also appealed people to buy coffee to the author.

I believe in this cause. I would love this project to be maintained and get support for php observability API.

For this, I need to know a little more information about the Author. All I know right now is name of the author is Sylvain L. 
A face to the name and a short interview would be excellent. I could not find any information online about the author. Couldnt find linkedin profiler either.

I would like to clarify, I don't want anything out of this except this repository to be maintained

I believe Author deserves a lot of recognition for this excellent gift to php community. 
Please help. Please reach out to jakhotiyaabhishek@gmail.com

**Comment by NoiseByNorthwest** [comment]
Hi Abhishek,

First of all thank you for this feedback and your willingness to promote the use of SPX at such scale !

> I believe in this cause. I would love this project to be maintained

It will be maintained as long as it is considered to be useful by many developers. I mean it is OSS so by me or even someone else.

> and get support for php observability API.

Why do you mean by "observability API", the internal observer API ? There is an issue here https://github.com/NoiseByNorthwest/php-spx/issues/215
The main reason it has not been done yet is not a lack of time & money but rather the lack of clue about how many users really need that.

> For this, I need to know a little more information about the Author. All I know right now is name of the author is Sylvain L.
A face to the name and a short interview would be excellent. I could not find any information online about the author. Couldnt find linkedin profiler either.

As I said above, there is no need for funding. But, I'm nevertheless open to answer to questions about me or the many whys and hows about this project as long as it sounds relevant for SPX users. By the way, my full name is simply in the license and my email in the git log of this project.

> I would like to clarify, I don't want anything out of this except this repository to be maintained

The more this project is recognized as being useful by many developers, the more its support is secured. I'm speaking for myself, I will not stop maintaining this project if it helps thousands of people. And money is not currently an issue since this is not a big & complex project.

> I believe Author deserves a lot of recognition for this excellent gift to php community.
Please help. Please reach out to jakhotiyaabhishek@gmail.com

Thank you very much for these words ! But I've to be honest, the growing popularity of SPX is already the kind of recognition I've expected from the beginning.
